config/environments/development.rb

```diff
@@ -48,7 +43,15 @@
   # routes, locales, etc. This feature depends on the listen gem.
   # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
 
+
+  # Action Mailer
+  config.action_mailer.perform_caching = false
   config.action_mailer.default_url_options = { host: 'localhost:3000' }
   config.action_mailer.raise_delivery_errors = false
   config.action_mailer.preview_path = "#{Rails.root}/lib/mailer_previews"
+
+  # with letter_opener
+  config.action_mailer.delivery_method = :letter_opener
+  config.action_mailer.perform_deliveries = true
+  config.active_job.queue_adapter = :inline
```

Comment:
I'd suggest to not do that 😉 It's just too far from reality if the real adapter is only used in production.

Depending on the adapter we're using the difference until an email is visible in the browser is only microseconds anyways. With Sidekiq it would be serializing to Redis and deserializing again which is fast enough.